### PR TITLE
Add documentation for organization name check

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
@zubair-arbi @awais786 @waheedahmed 

Relates to: https://github.com/edx/credentials/pull/98

### CODE REVIEW

**Comment by ahsan-ul-haq** [comment]
@srpearce please review

**Comment by zubair-arbi** [comment]
Change looks good 👍

File: docs/credentials_admin.rst [hunk 1]
```diff
@@ -119,3 +119,5 @@ The following image shows the ``ProgramCertificate`` model in the Django adminis
 .. image:: _static/images/program_certificate.png
     :width: 600px
     :alt: Populated program certificate model
+
+In ``ProgramCertificate`` if ``Use organization name`` is checked then organization full name will be displayed on the certificate else short name would be displayed.
```
Comment (srpearce) [inline docs/credentials_admin.rst]: Is this setting only available for XSeries Programs, or is it available for regular courses as well?

**Comment by srpearce** [comment]
I'm going to 👍  this PR, so you can go ahead and merge it. 

Note, though, that the docs for the edX Credentials service are moving to the edx-documentation repo at edx-documentation/en_us/install_operations/source/credentials. Because the Credentials service is still unavailable outside edX, the PR to add the docs is still open. So, to make further changes to the docs, please add commits to the [Credentials documentation pull request](https://github.com/edx/edx-documentation/pull/922).

 After this PR is merged, I'll submit a PR against this repo to make that clear.

File: docs/credentials_admin.rst (hunk 1 above)
Comment (ahsan-ul-haq) [inline docs/credentials_admin.rst]: This is currently only available for XSeries Programs.

**Pull request merged**